src/pages/ProfileView.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { FaDiscord, FaEnvelope, FaGithubSquare, FaLink, FaLinkedin, FaMapMarkerAlt, FaPencilAlt } from 'react-icons/fa';
+import {
+    BIOGRAPHY,
```

Comment:
I mean if making the import smaller was the whole point then I could just do "import * as UserMetadataConstants" but personally I like keeping the constant length down so that UserMetadataConstants.BIOGRAPHY doesn't take up a ton of space on a line for example.

Reply:
It's not a point I care all that strongly about tho so I can do whatever the people want lol